app/javascript/packages/verify-flow/forgot-password/forgot-password.tsx

```diff
@@ -0,0 +1,60 @@
+import { PageHeading } from '@18f/identity-components';
+import { t } from '@18f/identity-i18n';
+import { Button } from '@18f/identity-components';
+import { getAssetPath } from '@18f/identity-assets';
+
+interface ForgotPasswordProps {
+  goBack: () => void;
+}
+
+export function ForgotPassword({ goBack }: ForgotPasswordProps) {
+  const stepIndicator = document.getElementsByTagName('lg-step-indicator')[0];
+  hideStepIndicator();
+
+  function goToResetPassword() {
+    const resetPasswordUrl = `${window.location.origin}/forgot_password`;
```

Comment:
We might want to try pass this somehow from the server-side to the client, so that we can avoid issues if we were to ever change this path in the future (rely on `routes.rb` as the source of truth). We could do something similar to what we've done with `completion_url`, or maybe create an API that we can call to retrieve the URL, or maybe expand on "completion URL" to be able to provide some parameter which describes a completion scenario (e.g. `?reason=forgot_password`).